experiments/34x25/slotnick_etal_2001/uber_slotnick_pairedsquares_coordinate.m

```diff
@@ -17,3 +17,4 @@
 opts = [opts, 'force', true]; %, 'bandpass', bandpass]
 % Run sergent task by training on all images
 [trn, tst] = de_SimulatorUber('vanhateren/250', 'slotnick_etal_2001/paired-squares/coordinate', opts, args);
+tst.stats.rej.cc.anova.test.stats
```

Comment:
It's fine. It makes sense to only show that information if it is explicitly asked for.